iphone/iphone/Titanium_Prefix.pch

```diff
@@ -11,10 +11,10 @@
 #include "ThirdpartyNS.h"
 #import <AvailabilityInternal.h>
 
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000
-#define IS_XCODE_9 true
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+#define IS_XCODE_11 true
```

Comment:
Since this is a new constant i would propose to change the naming. This is not really a check for the Xcode version but rather for the version of the iOS SDK that is used to compile the code.

Although a specific version fo the SDK  usually is tied to a specific Xcode version, i find it more expressive to check for the SDK version. That way one doesn't have to remember the Xcode and iOS SDK version number coupling.

```suggestion
#define IS_IOS_SDK_13 true
```

Reply:
I agree with your naming convention. We have other flag IS_XCODE_10 as well. I think we should change naming for both at same time to avoid any confusion. I think creating an internal ticket to change constant name, will be good way to deal with this.

Reply:
Sounds good. Please change the name in this PR for the new constant already.